### ISSUE TITLE
Invalid aria attribute as aria-required="true" is defined for the paragraph present in the added feedback tiles

### ISSUE THREAD

**Issue by polatengin** [microsoft/vsts-extension-retrospectives#581]
Note: Same issue reproes throughout the application.

Impact on a Person with Disability (PWD):
Visually challenged users who rely on screen reader for navigation are facing difficulty as Invalid aria attribute as aria-required="true" is defined for the paragraph present in the added feedback tiles.

Test Environment:
OS: Windows 11 [Version 23H2 (OS Build 22631.2715)]
Browser: Edge Version (119.0.2151.97), Chrome Version (119.0.6045.200)
URL: [Work items - Boards (visualstudio.com)](https://worldwidelearning.visualstudio.com/MCAPS%20Accessibility/_workitems/myactivity/)
Tools: Accessibility insights for web

Repro Steps:
Step 1: Open MCAPS ADO in Edge/Chrome browser.
Step 2: Navigate to 'Retrospective' in left navigation and invoke it.
Step 3: Navigate to the collect tab and invoke it. Run AI.
Step 4: Verify whether Invalid aria attribute as aria-required="true" is defined for the paragraph present in the added feedback tiles or not.

Actual Result: 
Invalid aria attribute as aria-required="true" is defined for the paragraph present in the added feedback tiles.
Current Observation: 
1. While navigating in focus mode, when focus lands on the edit field, NVDA is narrating as "Feedback title is test. Click to edit. paragraph required."
2. While navigating in browse mode, when focus lands on the edit field, NVDA is narrating as "Clickable test".

Expected Result:
Valid aria attribute should be defined for the paragraph present in the added feedback tiles.
Ensure that: 
1. While navigating in focus mode, when focus lands on the edit field, NVDA should narrate as "Feedback title is test. Click to edit. Paragraph required".
2. While navigating in browse mode, when focus lands on the edit field, NVDA should narrate as "Feedback title is test. Click to edit. Paragraph required".

Suggestive fix
1.Remove 'aria-required="true"' attribute for
paragraph present in the added feedback tiles.

K4W:
Target application: Retrospectives - Boards - https://worldwidelearning.visualstudio.com/MCAPS%20Accessibility/_apps/hub/ms-devlabs.team-retrospectives.home

Element path: iframe;.feedbackItem > .document-card-wrapper > .ms-DocumentCard.mainItemCard.root-98 > .card-integral-part > .card-content > div:nth-child(2) > .editable-document-card-title > div > .editable-text-container > .editable-text[aria-required="true"]

Snippet: <p class="editable-text" tabindex="0" aria-label="Feedback title is test. Click to edit." aria-required="true">test</p>

How to fix:
ARIA attribute is not allowed: aria-required="true"

MAS Reference:
[MAS 4.1.2 – Name, Role, Value](https://aka.ms/MAS4.1.2)
 
WCAG Reference:
[https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html](https://nam06.safelinks.protection.outlook.com/?url=https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html&data=04%7c01%7cv-vshivanii%40microsoft.com%7c94edc582aca645e49e5008d8c9ce70dc%7c72f988bf86f141af91ab2d7cd011db47%7c0%7c0%7c637481235788101819%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=2zqGQGN70dzqnJ5k4IljdohWbT/9RGZnB3nk2fwx1o0%3D&reserved=0)

AzDO Reference Link:
https://worldwidelearning.visualstudio.com/MCAPS%20Accessibility/_workitems/edit/1713018
